filament/backend/src/vulkan/VulkanDriver.h

```diff
@@ -77,17 +77,6 @@ class VulkanDriver final : public DriverBase {
     VulkanDriver(VulkanDriver const&) = delete;
     VulkanDriver& operator = (VulkanDriver const&) = delete;
 
-    void purge() noexcept override {
-        // First we trigger garbage collection of Vulkan resources. This ensures that transient
-        // resources (e.g. the ReadPixels staging buffer) are removed if their refcount is 0, which
-        // allows related BufferDescriptors to move to the purge list.
-        mDisposer.gc();
```

Comment:
Does `mDispose.gc()` need to be called elsewhere now?

Reply:
The gc() func is already called in beginFrame().  I recently added a new call inside purge() in order to force the callback to trigger in unit tests that draw only one frame, but that was a wrong thing to do, thread-wise.

If the unit test issue comes up again then yeah we might need to add a call to gc() somewhere.  This PR is basically a partial revert.